pkg/stacker/base.go

```diff
@@ -46,7 +46,15 @@ func GetBase(o BaseLayerOpts) error {
 	case types.OCILayer:
 		fallthrough
 	case types.DockerLayer:
-		return importContainersImage(o.Layer.From, o.Config, o.Progress)
+		err := importContainersImage(o.Layer.From, o.Config, o.Progress)
+		if o.Layer.Bom != nil && o.Layer.Bom.Generate {
+			bomPath := path.Join(o.Config.StackerDir, "artifacts", o.Name)
+			err = getArtifact(bomPath, "application/spdx+json", o.Layer.From.Url, "", "", o.Layer.From.Insecure)
+			if err != nil {
+				log.Warnf("sbom for image %s not found", o.Layer.From.Url)
```

Comment:
"No SBOM" isn't really a warning.  If someone specified that a SBOM should be imported, then its an error.  If they didn't then its "operating as expected".

Can you turn that down to Info or even Debug?

Reply:
Let's make this an error.